modules/dnn/src/ie_ngraph.cpp

```diff
@@ -209,61 +219,34 @@ class InfEngineNgraphCustomLayer : public InferenceEngine::ILayerExecImpl
     }
 
 private:
-    InferenceEngine::CNNLayer cnnLayer;
+    std::shared_ptr<NgraphCustomOp> node;
     dnn::Layer* cvLayer;
     std::vector<Mat> internals;
 };
 
 
-class InfEngineNgraphCustomLayerFactory : public InferenceEngine::ILayerImplFactory {
```

Comment:
should not we put this under conditional compilation?

Reply:
Done, thanks!